apps/api/src/applicant/dto/ienapplicant-update.dto.ts

```diff
@@ -0,0 +1,90 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import {
+  IsDateString,
+  IsOptional,
+  IsString,
+  Length,
+  IsObject,
+  IsBoolean,
+  IsArray,
+} from 'class-validator';
+import { IENApplicantUpdateDTO } from '@ien/common';
+
+export class IENApplicantUpdateAPIDTO extends IENApplicantUpdateDTO {
```

Comment:
Duplicating all these DTO's in order to add documentation feels like an anti pattern, we should create a task to see if we can fix this. There should be a better way to do this.

Reply:
I'd share one link in our chat about this issue discuss on git, but was not able to resolve it in nextjs FE, It works fine with the decorator in nestjs. So I had put it as it is, If you get some time could you please take a look at it this or maybe next week.